napvig:
  rate: 100
  topics:
    subs:
      scan: /scan
      odom: /odom
      target: /target
    pubs:
      measures: /measures
      map_values: /map_values
      setpoint: /setpoint
      search_history: /search_history
  map:
    measure_radius: 0.2
    smooth_radius: 0.1
    precision: 100
    dimensions: 2
  napvig:
    step_ahead_size: 0.2
    gradient_step_size: 0.3
    termination_distance: 0.01
    termination_count: 50
    lookahead_horizon: 4
    algorithm: predict_collision # {predict_collision, single_step}
    min_distance: 0.15
    scatter_variance: 1 # rad
  map_test:
    range_min: -0.5
    range_max: 1.5
    range_step: 0.2
    draw: none   # {none, value, grad}
unicycle_control:
  rate: 100.0
  state_topic: /odom
  ref_topic: /setpoint
  command_topic: /cmd_vel
  gains: [0.6, 0.3, 0.03]